packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -197,7 +129,7 @@
                 }
             ],
             "message": {
-                "id": "e47cda88-3f39-455b-b4de-01c4270d16e5",
+                "id": "6f793f82-5e13-492f-803c-100c8dcc3cb6",
```

Comment:
By changing the message.id the message will be displayed again for the users who dismissed it, right? That sounds right to me in this case, just to confirm my understanding...

Reply:
Yes, exactly.